### DISCUSSION TITLE
Allow viewing scores adjacent to your highscore rank beyond top 50 on in-game leaderboard

### DISCUSSION THREAD

**Discussion by MTGPROD** [ppy/osu#18837] [Ideas] [answered]
On osu!stable the scoreboard shows your score compared to the 50 best scores. But what I propose, and which would be more satisfying is for example if our current score is ranked 2000, it would display the score of the 1999th etc. And that it goes up as you go along. You don't have to do 1 by 1 but by 50, 100 or 1000 depending on the number of scores recorded. Instead of just the 50 best scores !

(I think that could affect bancho's performances, so at this level idk if it's a good idea)

**Comment by respektive** [comment, marked as answer]
https://github.com/ppy/osu/discussions/17673
